app/templates/measure-concepts/measure-concept.hbs

```diff
@@ -25,7 +24,16 @@
         <LinkTo class="au-c-button" @route="measure-concepts.edit" @model={{@model.id}}>
           {{t "measureconcept.edit"}}
         </LinkTo>
-        <AuButton @alert={{true}} {{on "click" (fn this.removeMeasureConcept @model)}}>{{t "measureconcept.delete"}}</AuButton>
+        <AuButton @alert={{true}} {{on "click" (fn (mut this.popup) true) }}>{{t "measureconcept.delete"}}</AuButton>
```

Comment:
Personally, I would replace this with an action and a tracked property in the template. I don't like the mut helper very much and I think this is hard to read. But it's only a personal opinion, we can leave it like this, since it works (and we can always refactor if needed).

Reply:
I would make the property name more "booleany" though. Maybe something similar to `shouldShowConfirmation`? Something like that will make it more clear that it's a boolean value. Again, completely optional, it's just my personal naming convention.

Reply:
Thanks for the feedback. The implementation is more or less a copy paste from appuniversem. ModalState looks like an improvement though.